frontend/src/pages/Home/CheckoutList.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from "react";
+import BasePage from "../BasePage";
+import { Jumbotron, Nav, NavItem, NavLink, Row, Col, Button, Label, FormGroup} from 'reactstrap';
+import classnames from 'classnames';
+import axios from "../../axiosInstance";
+import StripedTable from "../../components/StripedTable";
+
+class CheckoutList extends Component {
+    constructor(props) {
+        super(props);
+        this.state = { 
+            loading: true, 
+            instruction: "",  
+            page_load: false, 
+            item_selected: false, 
+            categories: [], 
+            page_items: [],
+            page_pk: [],
+            pk: false,
```

Comment:
What are these two variables for? They don't seem to be used anywhere (same for `instruction`)